fe/src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import classNames from 'classnames';
+
+import heartLike from '../../assets/images/icons/favourite.svg';
+import logo from '../../assets/images/icons/logo.svg';
+import cross from '../../assets/images/icons/close.svg';
+import bag from '../../assets/images/icons/bag.svg';
+
+import { Link, NavLink } from 'react-router-dom';
+
+import './burger-menu.scss';
+
+export const BurgerMenu: React.FC = () => {
+  const [showBurgerMenu, setShowBurgerMenu] = useState(false);
+
+  const changeBurgerState = () => {
+    setShowBurgerMenu(true);
+  };
+
+  return (
+    <header
```

Comment:
```suggestion
    <div
```
the header tag should be used only for its intended purpose

Reply:
I used it because it is header. Another part of it